sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -264,10 +350,10 @@ class TopicDescription(object):  # pylint:disable=too-many-instance-attributes
     :type is_anonymous_accessible: bool
     :keyword authorization_rules: Authorization rules for resource.
     :type authorization_rules:
-     list[~azure.servicebus.management._generated.models.AuthorizationRule]
+     list[~azure.servicebus.management.AuthorizationRule]
     :keyword status: Status of a Service Bus resource. Possible values include: "Active", "Creating",
      "Deleting", "Disabled", "ReceiveDisabled", "Renaming", "Restoring", "SendDisabled", "Unknown".
-    :type status: str or ~azure.servicebus.management._generated.models.EntityStatus
+    :type status: str or ~azure.servicebus.management.models.EntityStatus
```

Comment:
namespace: `azure.servicebus.management.models.EntityStatus` or `azure.servicebus.management.EntityStatus`?

because line 353 says `azure.servicebus.management.AuthorizationRule` while here "models" package is mentioned.

Reply:
Good catch. corrected